src/components/new-safe/create/steps/StatusStep/StatusStepper.tsx

```diff
@@ -28,7 +28,7 @@ const StatusStepper = ({ pendingSafe, status }: { pendingSafe: PendingSafeData;
         </StatusStep>
       </Step>
       <Step>
-        <StatusStep isLoading={!pendingSafe.txHash} safeAddress={pendingSafe.safeAddress}>
+        <StatusStep isLoading={!pendingSafe.txHash && !pendingSafe.taskId} safeAddress={pendingSafe.safeAddress}>
```

Comment:
Wouldn't this break normal safe creations since there is never a `taskId`? Should it be `||` instead?

Reply:
Good catch. It was working well for relaying but would break in the normal Safe creations. Fixed it.